### PR TITLE
[RFR] Major updates

### PR DESCRIPTION
**Pull request by brikou**

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -1,22 +1,10 @@
...
-    "extends": "airbnb",
     "env": {
         "mocha": true,
     },
+    "extends": "airbnb",
     "rules": {
         "indent": [2, 4],
-    },
-    "globals": {
-      "after": false,
```
Comment (djhi) [inline .eslintrc]: why removing mocha globals ?

File: src/frontend/js/app/configureStore.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ export default function configureStore(reducers, routes, initialState) {
         applyMiddleware(
             thunkMiddleware,
             syncHistory(history),
-            createSagaMiddleware(sagas),
```
Comment (djhi) [inline src/frontend/js/app/configureStore.js]: Why removing commas ? This is (was ?) an airbnb best practice.

File: src/frontend/js/product/ProductItem.spec.js [hunk 3]
```diff
@@ -41,7 +40,7 @@ describe('Component ProductItem', () => {
         const wrapper = shallow(<ProductItem {...props} />);
         assert(
             wrapper.contains(<Link to={'/products/42'} className="card-link">Details</Link>),
-            'ProductItem should contains a link to specified product',
+            'ProductItem should contains a link to specified product'
```
Comment (djhi) [inline src/frontend/js/product/ProductItem.spec.js]: same

**Comment by djhi** [comment]
Nice !

File: src/frontend/js/product/ProductItem.spec.js (hunk 3 above)
Comment (Kmaschta) [inline src/frontend/js/product/ProductItem.spec.js]: Here, because it's a function and not an object, IMO

Comment (djhi) [inline src/frontend/js/product/ProductItem.spec.js]: doh

File: .eslintrc (hunk 1 above)
Comment (brikou) [inline .eslintrc]: Because it is already defined as global.

```json
 "mocha": true,
```

File: src/frontend/js/app/configureStore.js (hunk 2 above)
Comment (brikou) [inline src/frontend/js/app/configureStore.js]: It is not an array (and so is marked as errorneous).

File: src/frontend/js/product/ProductItem.spec.js (hunk 3 above)
Comment (brikou) [inline src/frontend/js/product/ProductItem.spec.js]: Same :)

**Comment by brikou** [comment]
I still need to upgrade those before RFR

```
history                             1.17.0         1.17.0   2.0.1  
react-router-redux                   3.0.0          3.0.0   4.0.0  
```

File: src/frontend/js/app/configureStore.js (hunk 2 above)
Comment (djhi) [inline src/frontend/js/app/configureStore.js]: Nvm I should read more carefully...

File: .eslintrc (hunk 1 above)
Comment (djhi) [inline .eslintrc]: Didn't know about this !

**Comment by brikou** [comment]
I've updated history, but upgrading `react-router-redux` requires a dedicated PR so switching to RFR ping @djhi @Kmaschta

**Pull request merged**